crypto/mem_sec.c

```diff
@@ -21,7 +21,7 @@
 #include <string.h>
 
 /* e_os.h includes unistd.h, which defines _POSIX_VERSION */
-#if defined(OPENSSL_SYS_UNIX) \
+#if !defined(OPENSSL_NO_SECURE_MEMORY) && defined(OPENSSL_SYS_UNIX) \
     && defined(_POSIX_VERSION) && _POSIX_VERSION >= 200112L
```

Comment:
oops, apparently I need to include unistd.h here, otherwise _POSIX_VERSION does not seem to be defined.